docker-compose.yml

```diff
@@ -101,6 +101,16 @@ services:
     ports:
       - "8000:8000"
 
+  web:
```

Comment:
Considering this is in a compose with a number of other web services, would it maybe make sense for the service to be called Cirrus?

Reply:
good call